### DISCUSSION TITLE
zigbbee2mqtt stops working randomly

### DISCUSSION THREAD

**Discussion by tixel1** [Koenkk/zigbee2mqtt#14929] [General]
It can take an hour or several days but it always stops working. Nothing comes out in the log and I have it in debug. You just see the last thing it did was a while ago. In other words, it doesn't give any error, it's just that if you see it 2 hours after it crashed, you see the last thing it did 2 hours ago. systemctl status says idle.
I have a zigstar v4 with the CC2652P on a raspberry with an extension cable.

**Comment by kalbfuss** [comment]
Just a thought: Is your power supply strong enough? It could be a brown out problem in connection with the serial port.

**Comment by tixel1** [comment]
It happens to me in a version installed with npm in /opt/zigbee2mqtt. and updated today. It just stops working but I don't see anything in the logs.
This is the latest in the logs:
debug 2022-11-21 17:27:39: Received Zigbee message from 'thermostat_down', type 'commandDataResponse', cluster 'manuSpecificYour', data '{"dpValues":[{"data":{"data":[1 ],"type":"Buffer"},"datatype":4,"dp":36}],"seq":35858}' from endpoint 1 with groupID 0
info 2022-11-21 17:27:39: MQTT publish: topic 'zigbee2mqtt/thermostat_down', payload '{"child_lock":"UNLOCK","current_heating_setpoint":19,"deadzone_temperature":null,"heat":" OFF","last_seen":"2022-11-21T16:27:39.833Z","linkquality":105,"local_temperature":18.5,"local_temperature_calibration":-3,"max_temperature":45,"max_temperature_limit":null ,"min_temperature_limit":null,"preset":"hold","preset_mode":"hold","program":[{"weekdays_p1":"6h:0m 20°C"},{"weekdays_p2":"11h :30m 21°C"},{"weekdays_p3":"13h:30m 22°C"},{"weekdays_p4":"17h:30m 23°C"},{"saturday_p1":"6h:0m 24°C "},{"saturday_p2":"12h:0m 23°C"},{"saturday_p3":"14h:30m 22°C"},{"saturday_p4":"17h:30m 21°C"},{" sunday_p1":"6h:0m 19°C"},{"sunday_p2":"12h:30m 20°C"},{"sunday_p3":"14h:30m 21°C"},{"sunday_p4":"18h :30m 20°C"}],"running_state":"idle","sensor":"AL","system_mode":"heat"}'
I have another version in a docker installed as a home assistant add-on and that doesn't seem to freeze. I have to give it more time.
